src/progress_bars/ProgressBars.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {classNames} from "../utils";
+
+// Props for ProgressBar component
+type ProgressBarProps = {
+    value?: number;
+    max?:number;
+    labeled?:boolean;
+    style?: any;
+    className?: string;
+    status?: ProgressBarStatus;
+    type?:ProgressBarType;
+    animation?:string[];
+};
+
+export enum ProgressBarType {
+    STATIC = "progress-static",
+    NORMAL = "progress",
+
+}
+
+export enum ProgressBarStatus {
+    SUCCESS = "success",
+    WARNING = "warning",
+    DANGER = "danger",
+}
+
+export const ProgressBarAnimation = {
+    LOOP: "loop",
+    FADE_OUT: "progress-fade",
+    FLASH: "flash",
+    FLASH_DANGER: "flash-danger",
+}
+
+export class ProgressBar extends React.PureComponent<ProgressBarProps> {
+
+    // By default Progress Bar will be normal progress bar not static
+    static defaultProps = {
+        type:ProgressBarType.NORMAL
+    }
+
+    render() {
+        const {
+               value,
+               max,
+               status,
+               type,
+               labeled,
+               style,
+               className,
+               animation
+            } = this.props;
+
+        // Label style for labled progress bar
+        const labelStyle = { display: "block"}
+
+        let Tag: any;
+        let labeledClassName;
+        let animationClasses;
+
+        if (labeled){
+            labeledClassName = 'labeled'
+        }
+
+        if (animation){
+            animationClasses = animation.join(' ')
+        }
+
+        Tag="div"
+
+        return (
+            <Tag
+                className={classNames([
+                    type,
+                    status,
+                    labeledClassName,
+                    animationClasses,
```

Comment:
It might be cleaner to have animation class names come in as a string (comma separated) instead of string[].  That way, you can just drop the property value here and skip the animation.join(' ') above.  That's the nice thing about using the classnames([1, 2, 3]) util function as it will take all comma separated values and format them for you.

Reply:
Removed animation prop and used className prop for same purpose.